src/internal/operations/backup.go

```diff
@@ -138,18 +138,28 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 		observe.Complete()
 	}()
 
+	ctx, flushMetrics := events.NewMetrics(ctx, logger.Writer{Ctx: ctx})
+	defer flushMetrics()
+
 	var runnable bool
 
 	// IsBackupRunnable checks if the user has services enabled to run a backup.
 	// it also checks for conditions like mailbox full.
 	runnable, err = op.bp.IsBackupRunnable(ctx, op.Selectors.PathService(), op.ResourceOwner.ID())
-	if err != nil || !runnable {
-		logger.CtxErr(ctx, err).Error("running backup")
-		op.Errors.Fail(clues.Wrap(err, "running backup"))
+	if err != nil {
+		logger.CtxErr(ctx, err).Error("verifying backup is runnable")
+		op.Errors.Fail(clues.Wrap(err, "verifying backup is runnable"))
+
+		return
 	}
 
-	ctx, flushMetrics := events.NewMetrics(ctx, logger.Writer{Ctx: ctx})
-	defer flushMetrics()
+	if !runnable {
+		err = clues.New("service unavailable")
+		logger.CtxErr(ctx, err).Error("verifying backup is runnable")
+		op.Errors.Fail(clues.Wrap(err, "verifying backup is runnable"))
+
+		return
```

Comment:
Different log and error messages between conditions help clarify the different cases we hit.  Also, `CtxErr` is for extracting structured error data out of an error that was produced by a child function.  There's no need to call when returning an ad-hoc error.  So I don't think we need the log here at all.
```suggestion
	if !runnable {
		op.Errors.Fail(clues.Wrap(graph.ErrServiceNotEnabled, "checking service enabled"))

		return
```